blackberggroup/VA-FELLOWSHIPS
# Feature 6125145790 change mary julius headshot

## What type of PR is this? (check all applicable)


## Description

Updated photo of Mary Julius on both the main Fellowships landing page (https://www.innovation.va.gov/ecosystem/views/programs/fellowship/) and her individual Fellowship page. 

## Related Tickets & Documents
- Monday Task #6125145790

## QA Instructions, Screenshots, Recordings

Check to see the updated image in both the home screen carousel and her Individual fellowship page.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/assets/img/fellows/mary-julius-sq.jpg
 - _site/assets/img/fellows/mary-julius.jpg
 - assets/img/fellows/mary-julius-sq.jpg
 - assets/img/fellows/mary-julius.jpg
